sycl/doc/extensions/proposed/sycl_ext_oneapi_complex_marray.asciidoc

```diff
@@ -0,0 +1,546 @@
+            friend marray operatorOP(marray &rhs) = delete;
+         
+            // OP is: &, |, ^
+            friend marray operatorOP(const marray &lhs, const marray &rhs) = delete;
+            friend marray operatorOP(const marray &lhs, const DataT &rhs) = delete;
+         
+            // OP is: &=, |=, ^=
+            friend marray &operatorOP(marray &lhs, const marray &rhs) = delete;
+            friend marray &operatorOP(marray &lhs, const DataT &rhs) = delete;
+         
+            // OP is: &&, ||
+            friend marray<bool, NumElements> operatorOP(const marray &lhs, const marray &rhs) = delete;
+            friend marray<bool, NumElements> operatorOP(const marray& lhs, const DataT &rhs) = delete;
+         
+            // OP is: <<, >>
+            friend marray operatorOP(const marray &lhs, const marray &rhs) = delete;
+            friend marray operatorOP(const marray &lhs, const DataT &rhs) = delete;
+         
+            // OP is: <<=, >>=
+            friend marray &operatorOP(marray &lhs, const marray &rhs) = delete;
+            friend marray &operatorOP(marray &lhs, const DataT &rhs) = delete;
+         
+            // OP is: ==, !=
+            friend marray<bool, NumElements> operatorOP(const marray &lhs, const marray &rhs) {
+            /* ... */ }
+            friend marray<bool, NumElements> operatorOP(const marray &lhs, const DataT &rhs) {
+            /* ... */ }
+
+            // OP is: <, >, <=, >=
+            friend marray<bool, NumElements> operatorOP(const marray &lhs, const marray &rhs) = delete;
+            friend marray<bool, NumElements> operatorOP(const marray &lhs, const DataT &rhs) = delete;
+         
+            friend marray operator~(const marray &v) = delete;
+         
+            // OP is: +, -, *, /
+            friend marray operatorOP(const DataT &lhs, const marray &rhs) { /* ... */ }
+
+            // OP is: %
+            friend marray operatorOP(const DataT &lhs, const marray &rhs) = delete;
+         
+            // OP is: &, |, ^
+            friend marray operatorOP(const DataT &lhs, const marray &rhs) = delete;
+         
+            // OP is: &&, ||
+            friend marray<bool, NumElements> operatorOP(const DataT &lhs, const marray &rhs) = delete;
+         
+            // OP is: <<, >>
+            friend marray operatorOP(const DataT &lhs, const marray &rhs) = delete;
+         
+            // OP is: ==, !=
+            friend marray<bool, NumElements> operatorOP(const DataT &lhs, const marray &rhs) {
+            /* ... */ }
+
+            // OP is: <, >, <=, >=
+            friend marray<bool, NumElements> operatorOP(const DataT &lhs, const marray &rhs)  = delete;
+         
+            friend marray<bool, NumElements> operator!(const marray &v) = delete;
+    }
+
+} // namespace oneapi
+} // namespace ext
+} // namespace sycl
+```
+
+The list of deleted operators are: %, %=, ++, --, +, -, &, |, ^, &=, |=, ^=,
+&&, ||, <<, >>, <<=, >>=, <, >, +<=+, >=, ~, !
```

Comment:
These operators are still not rendered correctly in HTML, but given my previous comment, I think these lines will just be deleted anyways.

Reply:
I have removed the operator list due to the above comment.